src/execution/service.ts

```diff
@@ -262,6 +262,15 @@ export class ServiceScriptExecution extends BaseExecutionWithCommand<ServiceScri
       entireExecutionAborted,
       adoptee,
     };
+    // Doing this here ensures that we always log when the
+    // service stops, no matter how that happens.
```

Comment:
Are there other conditions outside the `stopping` and `failing` states that could be missed otherwise?

I do like hanging this off the promise.

edit: is `child.kill({force: true});`  one of these additional cases that are now handled?

Reply:
Before this change we'd only log that it stopped if we stopped it outselves and it was in certain specific states. This way, no matter how it dies (e.g. it chooses to call process.exit()), we inform the logger about it.